example/index.js

```diff
@@ -0,0 +1,127 @@
+import { createLightNode } from "js-waku/lib/create_waku";
+import { waitForRemotePeer } from "js-waku/lib/wait_for_remote_peer";
+import * as noise from "@waku/js-noise";
+import QRCode from "qrcode";
+// TODO: Get rid of these
+import hexToArrayBuffer from "hex-to-array-buffer";
+import arrayBufferToHex from "array-buffer-to-hex";
+
+function getPairingInfo() {
```

Comment:
```suggestion
function getPairingInfofromUrl() {
```

Just to make it super clear